### PR TITLE
nautilus: build/ops: hidden corei7 requirement in binary packages

### PR DESCRIPTION
**Pull request by smithfarm**
https://tracker.ceph.com/issues/41350

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -267,11 +267,8 @@ if(WITH_BLUESTORE)
   endif()
 endif()
 
-if(CMAKE_SYSTEM_PROCESSOR MATCHES "i386|i686|amd64|x86_64|AMD64|aarch64")
-  option(WITH_SPDK "Enable SPDK" ON)
-else()
-  option(WITH_SPDK "Enable SPDK" OFF)
-endif()
+CMAKE_DEPENDENT_OPTION(WITH_SPDK "Enable SPDK" OFF
```
Comment (tchaikov) [inline CMakeLists.txt]:
```
-- Found aio: /usr/lib/x86_64-linux-gnu/libaio.so  
CMake Error at CMakeLists.txt:270 (CMAKE_DEPENDENT_OPTION):
  Unknown CMake command "CMAKE_DEPENDENT_OPTION".


-- Configuring incomplete, errors occurred!
```

i think we cannot backport 6fb738a. as we were not using a newer cmake by then.

**Comment by smithfarm** [comment]
@tchaikov Please take another look?

**Comment by smithfarm** [comment]
@tchaikov I believe I made the changes you requested. Could you check if your change request is still valid?

**Comment by yuriw** [comment]
[wip-yuri7-testing-2019-11-12-2103-nautilus](https://trello.com/c/WyvBSbE0/757-wip-yuri7-testing-2019-11-12-2103-nautilus)

**Pull request merged**